src/main/java/net/tiagofar78/prisonescape/game/prisonbuilding/Chest.java

```diff
@@ -67,7 +68,7 @@ public Inventory buildInventory() {
 		}
 		
 		for (int slot : _contentsIndexes) {
-			inv.setItem(slot, _contents.get(slot));
+			// TODO: Add to inventory
```

Comment:
Create a method in InventoryManager and add it here. Don't forget to pass as arguments a PrisonEscapePlayer and a PrisonEscapeItem.